spec/base_spec.rb

```diff
@@ -333,7 +333,7 @@ class DummyReplacedModel < DummyElasticSearchModel; end
         it "stores datetime in UTC" do
           time = Time.at(1_544_657_724)
           DummyElasticSearchModel.create!(my_string: "Hello", my_time: time)
-          expect(refresh_and_find_search_hit["_source"]).to eq(@default_fields.merge("my_time" => "2018-12-12T23:35:24Z"))
+          expect(refresh_and_find_search_hit["_source"]).to eq(@default_fields.merge("my_time" => "2018-12-12T23:35:24.000Z"))
```

Comment:
Similar to [Bobby's comment in the aggregate changes](https://github.com/Invoca/aggregate/pull/49#discussion_r413168333), would we have more peace of mind from using a time with milliseconds? (Same applies to the other test files as well)

Reply:
Done.